pandas/plotting/_core.py

```diff
@@ -2818,18 +2818,42 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw an area plot of the input series using matplotlib.
+
+        An area plot displays graphically quantitative data.
```

Comment:
It would be great to add some info about why this function is useful or some basic use cases. `Displays quantitative data visually` doesn't really give me very interesting information about what is the shape or usage of the plot.